combat-trainer.lic

```diff
@@ -213,6 +213,9 @@ class LootProcess
     @ritual_type = thanatology['ritual_type'].downcase
     echo("  @ritual_type: #{@ritual_type}") if $debug_mode_ct
 
+    @cycle_rituals = false
```

Comment:
Yeah, I think you are right here.  I initially wrote a second variable thinking that @ritual_type would change based on the ritual and I would lose track of the cycle status. But it doesn't appear to do that.  I'll make some changes based on this and see how they work.  Thank you!